etl-pipeline/tests/functional/mappings/test_publisher_backlist.py

```diff
@@ -0,0 +1,50 @@
+import json
+import os
+
+from digital_assets import get_stored_file_url
+from mappings.publisher_backlist import PublisherBacklistMapping
+from model import Part, FileFlags
+
+
+def test_map_pubclisher_backlist_record():
```

Comment:
nit: s/pubclisher/publisher

Also, I feel like this could be a unit test?

Reply:
ya this is one is tough since nothing is being mocked but we are functionally testing that the mapper works properly